lua/net.lua

```diff
@@ -20,7 +20,7 @@ function M.setup_block()
 
 	-- Change these parameters to configure the network.
 	local front_ports = {"enp133s0f0"}
-	local front_rx_queues = 2
+	local front_rx_queues = 3
```

Comment:
We don't need to change `back_tx_queues`.

I'll update our diagram so the GK-GT unit only has a receiving queue.